anaUltraScurve.py

```diff
@@ -122,6 +126,8 @@
 
 vSum  = ndict()
 vSum2 = ndict()
```

Comment:
This was probably myself who used these variable names and it was a bad idea in retrospect.  Can we change these to be a bit more descriptive?  At the very least we should be able to understand how `vSum2` is different from `vSum` here.